Bottom: 55b60791f5749981a1f83503b197e128b35380ac
Top:    f30524bb8868672300f7ccdd19e6d31167bbf949
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 09:50:30 -0500

Refresh of worker-configuration

---
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index 8837daef..2b484f64 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -5,13 +5,16 @@
 import os
 from buildbot.plugins import *
 
+c = WorkerConfig = {}
+
+
 DEFAULT_BBFLAGS = '-k'
 
 # Workers
 # The 'workers' list defines the set of recognized buildworkers. Each element is
 # a Worker object, specifying a unique worker name and password.  The same
 # worker name and password must be configured on the worker.
-workers = [
+c['workers'] = [
     worker.Worker("worker-ntel", "pass", max_builds=1),
 ]
 
@@ -19,7 +22,7 @@ DEFAULT_REPO = 'git@git.novatech-llc.com:ntel/setup-scripts.git'
 
 
 # CHANGESOURCES
-change_source = [
+c['change_source'] = [
     changes.GitPoller(
         repourl=DEFAULT_REPO,
         branches=['master', 'morty'],
@@ -28,7 +31,7 @@ change_source = [
 ]
 
 # SCHEDULERS
-schedulers = [
+c['schedulers'] = [
     schedulers.ForceScheduler(
         name="Force",
         label="Force NTEL OpenEmbedded Build",
@@ -117,7 +120,7 @@ schedulers = [
 # The 'builders' list defines the Builders, which tell Buildbot how to perform
 # a build: what steps, and which workers can execute them.  Note that any
 # particular build will only take place on one worker.
-builders = []
+c['builders'] = []
 
 git_lock = util.MasterLock("git")
 
@@ -265,7 +268,7 @@ class BitBakeFactory(util.BuildFactory):
         self.addStep(BitBakeArchive())
 
 
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="OrionLXm",
         name="ntel_orionlxm",
@@ -281,7 +284,7 @@ builders.append(
         }
     ))
 
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="Orion I/O",
         name="ntel_orion_io",
@@ -297,7 +300,7 @@ builders.append(
         }
     ))
 
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="OrionLX (CPX)",
         name="ntel_orionlx_cpx",
@@ -314,7 +317,7 @@ builders.append(
         }
     ))
 
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="OrionLX (Plus)",
         name="ntel_orionlx_plus",
@@ -330,7 +333,7 @@ builders.append(
         }
     ))
 
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="Orion (qemu)",
         name="ntel_qemux86_64",
@@ -347,7 +350,7 @@ builders.append(
     ))
 
 multiconfig = ['orionlx-cpx', 'orionlx-plus', 'orionlxm', 'orion-io']
-builders.append(
+c['builders'].append(
     util.BuilderConfig(
         description="Orion (all)",
         name="ntel_all",
